src/main/java/com/ldtteam/structurize/util/LanguageHandler.java

```diff
@@ -111,11 +108,6 @@ else if (object instanceof String)
             }
         }
 
-        if (translation == null)
```

Comment:
hmm?

Reply:
That was an IDEA suggestion and I was feeling carefree. Plus I like to see as little yellow squiggles as possible :)

Reply:
looks like this line <https://github.com/ldtteam/Structurize/pull/289/files/bbddcaeab9693ba5fc0597d030399f3dc32a4416#diff-6676addc4748aa15f73e5265e569a68c4c70246518b07987e045dafdd427edf1L67> should be > 0 instead of >=0

Reply:
Sorry that link doesn't redirect to a line, where are you talking about?

Reply:
https://github.com/ShadowProtocol/Structurize/blob/feature/datagen/src/main/java/com/ldtteam/structurize/util/LanguageHandler.java#L64

Reply:
Hard to tell what this segment is actually trying to achieve purely from the code, but after a bit of analysis I think it's fine as is.